corehq/apps/hqadmin/reports.py

```diff
@@ -101,7 +101,7 @@ class UserAuditReport(AdminReport, DatespanMixin):
 
     fields = [
         'corehq.apps.reports.filters.dates.DatespanFilter',
-        'corehq.apps.reports.filters.simple.SimpleUsername',
```

Comment:
why not modify or rename `SimpleUsername` rather than subclass it? It looks like `UserAuditReport` is the only report using this filter...

Reply:
right, but that filter has no purpose now, as its only purpose was for that report...Generally, I am not a fan of keeping around code that isn't being used.

Reply:
That's totally fair. I don't feel strongly either way. Changed in d692a1f94bc3c5dfcafb477b3136e1370089453f